mod/wikirate/set/right/activity.rb

```diff
@@ -37,7 +37,7 @@ def activity_item item
   end
 
   view :open do |args|
-    card.left.account ? super(args) : ''
+    card.left.present? && card.left.account ? super(args) : ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.